### DISCUSSION TITLE
Support for both 86 and 120 versions?

### DISCUSSION THREAD

**Discussion by e120guru** [Blackymas/NSPanel_HA_Blueprint#277] [General]
Does this support both the 86 and 120 versions of NSpanel?

**Comment by Blackymas** [comment]
you mean eu and us version?

**Comment by e120guru** [comment]
Yes... At first it seemed like they had different size screens - but when I look closer it seems they are just physically different, right?

**Reply by edwardtfn** [reply]
Screen size is slightly different: #713

**Comment by Blackymas** [comment]
i am already working on the us version. i think might come with release 3.1.0 this weekend.

US Landscape goes with he eu tft.

**Comment by albkk** [comment]
Do I understand it correctly? I can use the EU version to have landscape format on the US panel?

**Reply by albkk** [reply]
Ignore my question. I have seen the threads on landscape format for US version
